docs/dask.rst

```diff
@@ -92,6 +93,29 @@ installed.
 This can also be beneficial if you are using multiprocessing or multithreading to carry out calculations,
 because zarr works nicely with disk access from different threads and processes.
 
+Reading in CASA data and default chunk size
+-------------------------------------------
+
+CASA datasets are typically stored on disk with very small chunks - if we mapped these directly to
+dask array chunks, this would be very inefficient as the dask tree would then contain in some cases
```

Comment:
```suggestion
dask array chunks, this would be very inefficient as the [dask task graph](https://docs.dask.org/en/latest/graphs.html) would then contain in some cases
```
for consistent naming with the dask docs + easy reference